packages/cli/test/unit/commands/dashboard.unit.test.ts

```diff
@@ -0,0 +1,187 @@
+import {stdout, stderr} from 'stdout-stderr'
+import Cmd  from '../../../src/commands/dashboard'
+import runCommand from '../../helpers/runCommand'
+import * as nock from 'nock'
+import {expect} from 'chai'
+import {ago} from '../../../src/lib/time'
+import {unwrap} from '../../helpers/utils/unwrap'
+import os = require('os')
+
+describe('dashboard', function () {
+  if (os.platform() === 'win32') {
+    it('does not run on Windows', () => expect(true))
+    return
+  }
```

Comment:
Work around to avoid wrapping entire test in this:
https://github.com/heroku/cli/blob/main/packages/apps-v5/test/runtest.js#L3C30-L3C30